src/ethereum/ethNetwork.js

```diff
@@ -237,13 +242,16 @@ export class EthereumNetwork {
     )
   }
 
-  processEvmScanTransaction(tx: EvmScanTransaction, currencyCode: string) {
+  processEvmScanTransaction(
```

Comment:
I can't tell from this PR if the original issue of incorrect transaction amounts is resolved or if we're just reintroducing the problem.

Reply:
I could not reproduce the incorrect amounts with only Etherscan. I can’t verify Amberdata because our api key is maxed out. I can only assume amberdata may have been returning the bad info.